File: programming-basics-javascript/02a-conditional-statements-lab/07-area-of-figures.js
function areaOfFigures(input) {
  let figure = input[0];

  if (figure == "square") {
    side_a = Number(input[1]);
    area = side_a * side_a;
  } else if (figure == "rectangle") {
    side_a = Number(input[1]);
    side_b = Number(input[2]);
    area = side_a * side_b;
  } else if (figure == "circle") {
    radius = Number(input[1]);
    area = Math.PI * radius * radius;
  } else if (figure == "triangle") {
    side_a = Number(input[1]);
    height_a = Number(input[2]);
    area = side_a * height_a * 0.5;
  }
  console.log(area.toFixed(3));
}

File: programming-basics-python/02a-conditional-statements-lab/07_area_of_figures.py
from math import pi

figure = input()
area = 0

if figure == "square":
    side_a = float(input())
    area = side_a**2
    print(f"{area:.3f}")

elif figure == "rectangle":
    side_a = float(input())
    side_b = float(input())
    area = side_a * side_b
    print(f"{area:.3f}")
elif figure == "circle":
    side_a = float(input())
    area = side_a**2 * pi
    print(f"{area:.3f}")
elif figure == "triangle":
    side_a = float(input())
    h = float(input())
    area = (side_a * h) / 2
    print(f"{area:.3f}")
